azure-storage-blob/tests/blob/test_container.py

```diff
@@ -474,45 +470,61 @@ def test_set_container_acl_with_empty_signed_identifiers(self):
         self.assertIsNone(acl.public_access)
 
     @record
-    def test_set_container_acl_with_empty_signed_identifier(self):
+    def test_set_container_acl_with_empty_access_policy(self):
         # Arrange
         container = self._create_container()
+        identifier = SignedIdentifier(id='empty', access_policy=None)
+        identifiers = [identifier]
+        container.set_container_acl(identifiers)
 
         # Act
-        container.set_container_acl()
+        container.set_container_acl(identifiers)
```

Comment:
It seems like this line is called twice? (Also on 478)

Reply:
Thanks for catching this! It has been fixed in the next commit